fbw-a32nx/src/systems/instruments/src/EFB/Efb.tsx

```diff
@@ -73,9 +73,22 @@ interface BatteryStatus {
 export const usePower = () => React.useContext(PowerContext);
 
 // this returns either `A380_842` or `A320_251N` depending on the aircraft
-export const getAirframeType = () => new URL(
-    document.querySelectorAll('vcockpit-panel > *')[0].getAttribute('url'),
-).searchParams.get('Airframe');
+export function getAirframeType(): AircraftType {
+    const airframe = new URL(document.querySelectorAll('vcockpit-panel > *')[0].getAttribute('url')).searchParams.get('Airframe');
+    let airframeType: AircraftType;
+
+    switch (airframe) {
+    case 'A380_842':
+        airframeType = AircraftType.A380_842;
+        break;
+    case 'A320_251N':
+    default:
+        airframeType = AircraftType.A320_251N;
+        break;
+    }
+
+    return airframeType;
+}
```

Comment:
This could be completely removed and replaced by the new AircraftType enum/LVar
The existing method relies on a parameter in the panel.cfg for the EFB instrument which is not ideal.

Reply:
I think the URL param approach is a good one. It has less complications than a persistent Lvar and cannot be messed with at runtime by shared cockpit and replay tools.

Reply:
It requires a parameter in panel.cfg which is rather annoying and bound to one instrument. If at all it then at least not be the EFB but extra or systems host.

Reply:
It would be easy enough to put it just on the extras-host.

Reply:
moved over to extras-host as own module airframeCheck, still need a decision if the initialize of the aircraft type should be happend by url param or if it should be hardcoded.